router/src/main/resources/envoy.yaml.template

```diff
@@ -0,0 +1,157 @@
+admin:
```

Comment:
I think we need the license header here.

Reply:
Done with https://github.com/wso2/product-microgateway/pull/3263/commits/af1183fd784cf04f92ea422af82393b8772b823e